composer.json

```diff
@@ -12,13 +12,18 @@
     "require": {
         "php": "^7.3",
         "nexylan/slack": "^3.0",
+        "php-http/discovery": "^1.6",
         "php-http/httplug-bundle": "^1.17",
+        "php-http/httplug": "^2.0",
```

Comment:
The original "bug" fix: needed to force the httplug install to use the Psr interfaces, which v3 of `nexylan/slack` uses in `Client`.